data/abixen-platform/misuses/1/misuse.yml

```diff
@@ -0,0 +1,15 @@
+api:
+- javax.crypto.spec.IvParameterSpec
+violations:
+- insecure/condition/randomization
+description: >
+  First parameter while initializing the IvParameterSpec object was not properly randomized.
```

Comment:
"not properly randomized" is a fairly vague description of the problem. Can you provide more details, e.g., by explaining how it should be done/could be fixed?

Please us backticks to properly format `type names`.